services/src/main/java/org/keycloak/protocol/oidc/endpoints/TokenRevocationEndpoint.java

```diff
@@ -257,9 +258,9 @@ private void revokeClient() {
     }
 
     private void revokeAccessToken() {
-        TokenRevocationStoreProvider revocationStore = session.getProvider(TokenRevocationStoreProvider.class);
+        SingleUseTokenStoreProvider singleUseTokenStore = session.getProvider(SingleUseTokenStoreProvider.class);
         int currentTime = Time.currentTime();
         long lifespanInSecs = Math.max(token.getExp() - currentTime, 10);
-        revocationStore.putRevokedToken(token.getId(), lifespanInSecs);
+        singleUseTokenStore.put(token.getId(), lifespanInSecs, new HashMap<>());
```

Comment:
Would it be perhaps better to use some unique suffix here?
```suggestion
        singleUseTokenStore.put(token.getId() + ".revoked", lifespanInSecs, Collections.emptyMap());
```

Reply:
Thanks for suggestion. Added to prevent clashes in the shared storage with other usages (even future ones).